shoutem.push-notifications/app/middleware.js

```diff
@@ -0,0 +1,41 @@
+import _ from 'lodash';
+
+import { Alert } from 'react-native';
+
+import { SHOW_PUSH_NOTIFICATION } from './redux';
+
+import FCM from 'react-native-fcm';
+
+import { I18n } from 'shoutem.i18n';
+
+// eslint-disable-next-line no-unused-vars
+const showNotification = store => next => action => {  
+  if (action.type === SHOW_PUSH_NOTIFICATION) {
```

Comment:
This if is adding unnecessary nesting. Because this middleware only handles one action invert the code with:
```
if (action.type !== SHOW_PUSH_NOTIFICATION) {
  return next(action);
}
```
and reduce nesting.